app/models/concerns/csv_exporter.rb

```diff
@@ -0,0 +1,17 @@
+require 'csv'
+
+module CsvExporter
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def to_csv(columns: column_names, csv_options: {})
+      CSV.generate(csv_options) do |csv|
+        csv << columns.map(&:to_s)
```

Comment:
`column_names` is an activerecord method that gives a list of all the column names, and it is used to provide the default value for the `columns` kwarg if no other value is passed.